### PR TITLE
First pass at download page

### PR DESCRIPTION
**Pull request by NicholasAntonov**
not all edge cases tested, but this is 90% of the way there, good for review

### CODE REVIEW

File: settings/common.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
 MEDIA_URL = '/media/'
 
 # set once instead of in all the templates
-JQUERY_HOME = "/static/js/jquery-1.7.1.min.js"
+JQUERY_HOME = "/static/js/jquery-1.12.4.min.js"
 JQUERY_UI_HOME = "/static/js/jquery-ui-1.8.16.custom.min.js"
```
Comment (eshellman) [inline settings/common.py]: is jquery-ui up to date?

**Comment by eshellman** [comment]
download-desktop-prod
download-desktup

Reviewing the desktop view, because it needs a lot more work than the smartphone view.

1. The font sizes are all over the place- very uncomfortable to read.
2. The the actual download button is comparatively tiny, and inexplicable. For accessability, it must have alt text. Consider having a larger click area.
3. The download page has room for additional information like license, pdf icon, and source of the file host.
4. By comparison, the Set-up Kindle button is HUGE.
5. we don't need 766 pixels for the donation amount; our audience isn't quadrillionaires. give it a max-size. Maybe do that in django form?
6. The current design group areas of the page functionally. so for example the text "You'll need an unglue.it account to Send to Kindle." is grouped with the "Send to Kindle" button, and "Make this contribution" is grouped with the thank you button. in the responsive design, these associations are lost and the resulting UI is hard to figure out.

**Comment by eshellman** [comment]
"Save to Dropbox" doesn't work on a non-public server, but it opens a window on my iphone simulator, which I don't think is the right thing to do. Should check the dropbox documentation. Also the button is tiny.

File: frontend/templates/download.html [hunk 2]
```diff
@@ -91,14 +91,14 @@
 
         {% else %}
         <div class="download-body">
-            <h1>Downloads for <I><a href="{% url 'work' work.id %}">{{ work.title }}</a></i></h1>
+            <h1 class="h2">Downloads for <I><a href="{% url 'work' work.id %}">{{ work.title }}</a></i></h1>
```
Comment (eshellman) [inline frontend/templates/download.html]: I guess making h1 look right is too hard?

Comment (NicholasAntonov) [reply frontend/templates/download.html]: No, I still think that the h1 stying is fine, but just not for this page where we want to have low size text before the first header. I want to keep the <h1> tag for semantic and a11y reasons

**Comment by eshellman** [comment]
better, but desktop is still a problem.
dorian-prod
dorian-desktop

look at the relative real estate taken up by the actions that we want the user to take on the page: download or read, versus the back link to the work page.

File: frontend/templates/download.html (hunk 2 above)
Comment (eshellman) [reply frontend/templates/download.html]: I think the h1 styling is always too large on desktop (looks fine on mobile

**Comment by eshellman** [comment]
This is an improvement.

1. The "one-click" option should be at the top it it's available.
2. I don't like the 2xN layout on larger screens. Better to keep the 1xN layout because it makes it easier to compare options. Ideally the page should present fewer options, but I can work on that outside of the scope of your project.
3. in the 1xN layout, distributing the components horizontally makes them harder to read or activate. Pretend they're part of a sentence that belongs together.
4. In this layout, I'm wondering at the the duplication in the top headings 
> Downloads for The Picture of Dorian Gray
> Download freely available editions
This is partly a result of moving "1-click options" down on the page, but maybe something should be reworded.

**Comment by eshellman** [comment]
The "say thanks" button is broken. Have you inserted some sort of client-side validation?

Also, why is this button blue on my iphone and green on my desktop?

The header does aweful things when I turn from landscape to portrait, but I assume that's a site-wide issue.

**Comment by eshellman** [comment]
I don't have any experience using flexbox. How do you get the three items in a row to act like a single block, that get centered, rather than 3 blocks which get centered in three columns?

**Comment by NicholasAntonov** [comment]
Not quite sure I understand what you mean by that, though you might want to check out how justify-content works:



The items widths are all taken into account when positioning, and centering in this case, is determined

**Comment by eshellman** [comment]
There are a number of unresolved issues with the download page (and the header). Where should we track these?

**Comment by NicholasAntonov** [comment]
GitHub issues with a redesign tag seems like a good fit

**Pull request merged**